application-admintools-ui/src/main/resources/AdminTools/Code/HealthCheckJob.xml

```diff
@@ -45,7 +45,7 @@
   #healthCheckOutputJobStatus($jobStatus)
 #else
   $response.setStatus(404)
-  #error($escapetool.xml($services.localization.render("admintools.job.notFound")))
+  {{error}}($escapetool.xml($services.localization.render("admintools.job.notFound"))){{/error}}
```

Comment:
``$escapetool.xml()`` makes sense when you are in an HTML/XML context, i.e. inside the HTML macro, but I don't see any HTML macro here.

```suggestion
  {{error}}{{translation key="admintools.job.notFound"/}}{{/error}}
```